core/src/main/java/org/axonframework/queryhandling/SimpleQueryBus.java

```diff
@@ -188,6 +217,120 @@ public SimpleQueryBus(MessageMonitor<? super QueryMessage<?, ?>> messageMonitor,
                        .filter(Objects::nonNull);
     }
 
+    @SuppressWarnings("unchecked")
+    @Override
+    public <Q, I, U> Registration subscriptionQuery(SubscriptionQueryMessage<Q, I, U> query,
+                                                    UpdateHandler<I, U> updateHandler) {
+        MessageMonitor.MonitorCallback monitorCallback = messageMonitor.onMessageIngested(query);
+        SubscriptionQueryMessage<Q, I, U> interceptedQuery = intercept(query);
+        List<QuerySubscription> subs = subscriptions
+                .computeIfAbsent(interceptedQuery.getQueryName(), k -> new CopyOnWriteArrayList<>())
+                .stream()
+                .filter(subscription -> interceptedQuery.getResponseType().matches(subscription.getResponseType()))
+                .collect(Collectors.toList());
+        Registration registration = () -> true;
```

Comment:
We should return `false` in this case, because the subscription query is already (automatically) closed by the query handler.